definitions/tools/check_strand.cwl

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env cwl-runner
```

Comment:
I might suggest renaming this tool to "check_rna_strandedness", just to make it's purpose even clearer.